### ISSUE TITLE
[BUG] Mini+ Stock Kit "TEMP NOT MATCHING" Error

### ISSUE THREAD

**Issue by usaf-barnes** [prusa3d/Prusa-Firmware-Buddy#2993]
**Printer type** - [MINI]

**Printer firmware version** - [4.4.0+4180]

**Original or Custom firmware** - [Original]

**Optional upgrades** - [Texture Build Plate]

**USB drive or USB/Octoprint** - [USB]

**Describe the bug**
  Original Prusa Mini+ Error:  The screen displays the following text "TEMP NOT MATCHING", "Measured temperature is not matching expected value. Check the thermistor is in contact with hotend. In case of damage, replace it."

**How to reproduce**
  I placed the hotend temperature at 240 degrees and only after a minute the error message is shown.  This happens each time I attempt to print or test the thermistor.

**Expected behavior**
  The expected behavior is the hotend heats to set temperature and does not shut off.

**G-code**
  M104 S240.

**Crash dump file**
  Crash dump file saved and attached as a .docx file for uploading only.

  Here are the console output messages:
Send: M104 S240
Recv: ok
Send: M105
Recv: ok T:229.56 /240.00 B:24.80 /0.00 A:-30.00 /0.00 @:99 B@:0
Send: M105
Recv: ok T:229.37 /240.00 B:24.84 /0.00 A:-30.00 /0.00 @:101 B@:0
Send: M105
Recv: ok T:229.19 /240.00 B:25.03 /0.00 A:-30.00 /0.00 @:104 B@:0
Send: M105
Recv: ok T:229.75 /240.00 B:25.00 /0.00 A:-30.00 /0.00 @:61 B@:0
Send: M105
Recv: ok T:229.75 /240.00 B:24.92 /0.00 A:-30.00 /0.00 @:75 B@:0
Send: M105
Recv: ok T:230.23 /240.00 B:24.84 /0.00 A:-30.00 /0.00 @:82 B@:0
Send: M105
Recv: ok T:230.78 /240.00 B:25.03 /0.00 A:-30.00 /0.00 @:84 B@:0
Send: M105
Recv: ok T:231.09 /240.00 B:24.88 /0.00 A:-30.00 /0.00 @:88 B@:0
Send: M105
Recv: ok T:231.64 /240.00 B:25.06 /0.00 A:-30.00 /0.00 @:88 B@:0
Send: M105
Recv: ok T:232.03 /240.00 B:24.96 /0.00 A:-30.00 /0.00 @:89 B@:0
Send: M105
Recv: ok T:232.34 /240.00 B:24.96 /0.00 A:-30.00 /0.00 @:91 B@:0
Send: M105
Recv: ok T:232.97 /240.00 B:25.06 /0.00 A:-30.00 /0.00 @:91 B@:0
Send: M105
Recv: ok T:233.36 /240.00 B:25.06 /0.00 A:-30.00 /0.00 @:92 B@:0
Send: M105
Recv: ok T:233.59 /240.00 B:25.11 /0.00 A:-30.00 /0.00 @:95 B@:0
Send: M105
Recv: ok T:233.83 /240.00 B:24.96 /0.00 A:-30.00 /0.00 @:97 B@:0
Send: M105
Recv: ok T:234.45 /240.00 B:24.96 /0.00 A:-30.00 /0.00 @:96 B@:0
Send: M105
Recv: ok T:234.77 /240.00 B:25.00 /0.00 A:-30.00 /0.00 @:97 B@:0
Send: M105
Recv: ok T:235.16 /240.00 B:24.96 /0.00 A:-30.00 /0.00 @:98 B@:0
Send: M105
Recv: ok T:235.62 /240.00 B:25.06 /0.00 A:-30.00 /0.00 @:98 B@:0
Send: M105
Recv: ok T:235.86 /240.00 B:25.06 /0.00 A:-30.00 /0.00 @:100 B@:0
Send: M105
Recv: ok T:236.02 /240.00 B:25.06 /0.00 A:-30.00 /0.00 @:102 B@:0
Send: M105
Recv: ok T:236.56 /240.00 B:24.92 /0.00 A:-30.00 /0.00 @:101 B@:0
Send: M105
Recv: ok T:237.11 /240.00 B:25.06 /0.00 A:-30.00 /0.00 @:100 B@:0
Send: M105
Recv: ok T:237.19 /240.00 B:24.96 /0.00 A:-30.00 /0.00 @:102 B@:0
Send: M105
Recv: ok T:237.34 /240.00 B:25.09 /0.00 A:-30.00 /0.00 @:104 B@:0
Send: M105
Recv: ok T:237.58 /240.00 B:25.03 /0.00 A:-30.00 /0.00 @:104 B@:0
Send: M105
Recv: ok T:238.05 /240.00 B:25.03 /0.00 A:-30.00 /0.00 @:103 B@:0
Send: M105
Recv: ok T:238.52 /240.00 B:25.03 /0.00 A:-30.00 /0.00 @:102 B@:0
Send: M105
Recv: ok T:238.67 /240.00 B:25.03 /0.00 A:-30.00 /0.00 @:102 B@:0
Send: M105
Recv: ok T:238.98 /240.00 B:25.06 /0.00 A:-30.00 /0.00 @:102 B@:0
Send: M105
Recv: ok T:239.06 /0.00 B:25.03 /0.00 A:-30.00 /0.00 @:0 B@:0
Send: M105
Recv: ok T:239.53 /0.00 B:25.14 /0.00 A:-30.00 /0.00 @:0 B@:0
Send: M105
Recv: ok T:238.67 /0.00 B:25.09 /0.00 A:-30.00 /0.00 @:0 B@:0
Send: M105
Recv: ok T:237.50 /0.00 B:25.03 /0.00 A:-30.00 /0.00 @:0 B@:0
Send: M105
Recv: ok T:235.62 /0.00 B:25.17 /0.00 A:-30.00 /0.00 @:0 B@:0
Send: M105
Recv: ok T:233.59 /0.00 B:25.23 /0.00 A:-30.00 /0.00 @:0 B@:0

**Video**
  No video at this time.
[dump.bin.docx](https://github.com/prusa3d/Prusa-Firmware-Buddy/files/10726956/dump.bin.docx)

**Comment by usaf-barnes** [comment]
Upgraded to FW 4.4.1 and the problem has persisted.

**Comment by Prusa-Support** [comment]
Hello, thank you for the report. It is possible it is not a bug, as the temperature was indeed very close but not matching from the serial log - as a first check, can you please send me pictures of hotend, from all sides, especially where hotend heater and hotend thermistor join the heater block? Do you have a multimeter? I'll expand on some measurements with multimeter, in an attempt to catch a fault in one of the components. 

Feel free to contact me directly, info@prusa3d.com mentioning me, we can review it from there - I'll report it to devs in case we'd confirm it to be a bug, though it looks like a hardware issue. 

Alessandro Pantaleo
Prusa Research

**Comment by ricoshe81x** [comment]
Hi, I have the very same problem. It occurred since I updated my firmware to 4.4.1. Before that I never had that issue! I've already purchased and changed the thermistor & heater (purchased from purse of course!).

**Comment by Prusa-Support** [comment]
Hello, thank you for the report, noted for devs. @usaf-barnes, @ricoshe81x, if you have a multimeter, can you please measure, with **printer off and cold**, the resistance of the hotend thermistor, and hotend heater? This article covers how to do the measurement: https://help.prusa3d.com/article/multimeter-usage_2117, in case of any doubts, feel free to reach out to me info@prusa3d.com mentioning me, I can expand on each point. 

Feel free to send me pictures of the hotend from several angles, especially below heater block, and where hotend heater and hotend thermistor join the heater block - here or via email to me. That way we'd have some more info on the situation. 

Alessandro Pantaleo
Prusa Research

**Comment by usaf-barnes** [comment]
Hello Alessandro,

I have uploaded several items to imgur.  The first link if for the submission for the help desk and the second are the readings you have requested.

https://imgur.com/a/qdlaWWM

https://imgur.com/a/ohxoPez

I appreciate the assistance!
-Chris Barnes

**Comment by Prusa-Support** [comment]
Please double-check the thermistor and heater alignment.
The [heater](https://help.prusa3d.com/guide/how-to-replace-a-hotend-heater-mini-mini_155068#155604) needs to be well-centered.
The [thermistor](https://help.prusa3d.com/guide/how-to-replace-a-hotend-thermistor-mini-mini_147001#147417) needs to be flush on the right.

Please allow me to close this issue as a duplicate of https://github.com/prusa3d/Prusa-Firmware-Buddy/issues/2768.

Michele Moramarco
Prusa Research

**Comment by usaf-barnes** [comment]
It was found to be a crushed thermistor, my apologies for jumping to the bug.

**Comment by usaf-barnes** [comment]
Image has been posted on imgur - https://imgur.com/a/LwXlZYv

**Comment by Prusa-Support** [comment]
Hello, thank you for the update and for the pic, no problem at all, and good we checked it - in case your printer is within warranty, can you please reach me via email (info@prusa3d.com mentioning me in the subject), with all the details of the printer? We can review it in more detail from there. 

Alessandro Pantaleo
Prusa Research
